### PR TITLE
[1105] Edit Well Location

### PR DESCRIPTION
**Pull request by Sybrand**
Tests added
Map coordinate forced to numbers

### CODE REVIEW

File: app/frontend/src/submissions/components/SubmissionForm/CoordsMap.vue [hunk 1]
```diff
@@ -37,6 +38,11 @@ export default {
       map: null
     }
   },
+  beforeCreate () {
+    // If lat/long aren't numbers, try to make them numbers.
+    this.$options.propsData['latitude'] = Number(this.$options.propsData['latitude'])
+    this.$options.propsData['longitude'] = Number(this.$options.propsData['longitude'])
+  },
```
Comment (stephenhillier) [inline app/frontend/src/submissions/components/SubmissionForm/CoordsMap.vue]: could this be in the parent component instead? 
```<coords-map :latitude="Number(latitude)">```

Comment (Sybrand) [reply app/frontend/src/submissions/components/SubmissionForm/CoordsMap.vue]: That's only a million times better!

Comment (Sybrand) [reply app/frontend/src/submissions/components/SubmissionForm/CoordsMap.vue]: Yes -  that's 10^6 time better.

**Comment by garywong-bc** [comment]
FYI in a subsequent phase we will be loading into well.geom which will be a PostGIS spatial column.

**Pull request merged**